### DISCUSSION TITLE
Microphone; Audiomoth or EMT 2

### DISCUSSION THREAD

**Discussion by MueJosh** [rdz-oss/BattyBirdNET-Pi#34] [Q&A] [answered]
EDIT: I've just ordered two Audiomoth 1.2.0 - the question remains the same (might be interesting for other users)

Hey, 

Id like to build a BattyBirdNET-Pi but I don't have a microphone. 
There are 3 mentioned mics- the Audiomoth (1.2.X via USB; the usb version is not available for now) for around 150€ including shipping (+tax), the wildlifeacoustics Echo meter Touch 2 for around 240€ (sold from germany) or the pro variant for ~420€. 

Is the EMT 2 (+pro) substantially better compared to the Audiomoth? (Im in europe, i dont need to detect other bats) 
~~Is the price really worth it? ~~

Considering that i normally solder my mics (e.g.  for the birdnetpi) and pay around 6€ including the soundcard, thats a BIG amount of money... 

Im doing this for personal use and just for fun, i just like to know whats flying in my garden.

Thanks! 

Have a nice day

Joshua

**Comment by rdz-oss** [comment, marked as answer]
Hi @MueJosh ,

it really depends on a persons preferences and requirements. For me, I want an inexpensive mic that can record bats and birds. The bats in my area can go up to maybe 130kHz so that should be detectable in the upper end. I want to run it 24/7 and need a protective casing.

At this time my favorite is the Audiomoth v1.2 in USB version. The Wildlife acoustics models struggle more with the background noise from the electric surroundings. The audiomoth is noisier, but kind of covers that type of electrical background noise up with its own noise. It works well for detecting the local bats as well as birds. Also, it comes with a good protective casing and I can buy almost two for the price of one EMT 2.

I do enjoy the EMT 2 models, however, yet for a different purpose. I like to use them on the go with my android phone. I have the phone with me anyway to adding an EMT 2 allows me to ID bats on the go/spontaneously. Great for that together with their app.

The EMT 2 pro has significantly less noise than the audiomoth or regular EMT 2. If you want to have clear recordings this would be the go to. It also records the noises made by the Pi, transformer etc well. So you will have to filter out this electrical background noise in some way. or they will be prominent in the recordings. 

The pro makes sense if you require low noise to signal ratio and a good resolution at the upper end of a call around say 180kHz to differentiate e.g. between Myotis species or if you record horsehoe bats. That would be the case if you do a professional survey or live close to a horsehoe bat population. Other users will not have a large benefit from the added 'pro' capabilities.

So at this time I use the audiomoth v1.2 usb with casing for the BattyBirdNET-Pi, the EMT 2 on hikes for spontaneous ID of bats and the EMT 2 pro if I need to make the best possible recordings e.g. for the machine learning data set or to ID a tricky bat species. The audiomoth v1.2 with the battery pack and that casing is great to leave it for three days/nights at a place and the check the recordings for bats with e.g the BattyBirdNET-Analyzer.

There are likely more options from e.g. dodotronic that you could check out, or the PiPipistrelle which you can solder yourself.

That for a quick answer,

Richard

**Comment by TF-Ingolstadt** [comment]
Hello, i just want to add that i was not able to get the EMT 2 pro with firmware 2.0 running on my BattyBirdNET-Pi. It seems that at least since firmware 2.0 it does not behave like a standard USB microphone. Connected to the Raspberry pi, the Audiomoth 1.2.0 works fine.

Regarding the EMT 2 pro i agree with rdz-oss. It is simply built for another use case and unfortunately not compatible (anymore?) with Raspberry. Not cheap but if you consider that you get a mobile bat recorder with real time auto id i think it is still well worth the money.  
Best regards, Thomas

**Comment by NathanLitjens** [comment]
I'm having the issue with the EMT2Pro not being recognised at all by the installation. I have selected it in the advanced settings under "noise profile"

The emt2p is working when plugged in to my tablet, so it's not faulty. 

Do I need to install something else to get it to run?
